File: c++/class/person.cpp
#include<iostream>
#include <string>
using namespace std;

class Person
{
	 public:

			// 共有
			void setName(string strname){
				 name = strname;
			}
			string Pr_name(){
				 return name;
			}
	 private:
			// 私有
			string name;

};
int main(int argc,char *argv[])
{
	 Person p;
	 p.setName("张三");
	 cout << "姓名: " << p.Pr_name() << endl;
	 return 0;
}

File: java/temp/temp/Computer/src/person.java
public class person {
    private String name;
    private int age;
    public person(String name,int age) {
        this.name  = name;
        this.age = age;
    }
    public String say() {
        return "姓名: " + name + "年龄: " + age;
    }

}
